charts/azure-k8s-metrics-adapter/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
```

Comment:
make description one-liner from the readme: "An implementation of the Kubernetes Custom Metrics API and External Metrics API for Azure Services. "

Reply:
Will do